wgpu-hal/src/metal/command.rs

```diff
@@ -187,6 +187,14 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
     ) where
         T: Iterator<Item = crate::TextureCopy>,
     {
+        let dst_texture = if src.format != dst.format {
+            let raw_format = self.shared.private_caps.map_format(src.format);
+            Cow::Owned(objc::rc::autoreleasepool(|| {
```

Comment:
The autorelease pool will cover the destruction of this texture view?

Reply:
As far as I understand, yeah. But I'm not too familiar with the rust <-> objective-c interaction.